index.html

```diff
@@ -2022,11 +2022,11 @@ <h3>Securing Verifiable Credentials</h3>
         </p>
         <p>
 This specification recognizes two classes of securing mechanisms: those that use
-external proofs and those that use embedded proofs. An
-<dfn class="export">external proof</dfn> is one that wraps an expression of
+enveloping proofs and those that use watermarking proofs. An
```

Comment:
watermarking typically implies some difficulty in removing the mark... this is not true of embedded proofs.

I think the term embedded proofs was better here...